vector-im/element-android
# [Session manager] Hide push notification toggle when there is no server support (PSG-970)

## Type of change


## Content


Checking the support for MC3881 about new enabled field for Pusher to render the push notification toggle for a session in the overview screen.

## Motivation and context


Closes #7457 

## Screenshots / GIFs





## Tests



- Signin to matrix server
- Enable the new session manager labs flag
- Go to Settings -> Security & Privacy -> Show all sessions
- Go to a mobile session overview
- Check the push notification toggle is not displayed (matrix does not support the feature yet)
- Go to a web session overview
- Check the push notification toggle is displayed (since there should be an associated account data setting)
- Signin to a server which supports MCS3881
- Go to Settings -> Security & Privacy -> Show all sessions
- Go to a mobile session overview
- Check the push notification toggle is displayed

## Tested devices

- OS version(s): Android 11

## Checklist
